docs/ios/deployment/overview.md

```diff
@@ -0,0 +1,193 @@
+- A Mac on which you can build your app.
+
+The process for publishing a .NET MAUI iOS app is as follows:
+
+1. Create a certificate signing request on your Mac.
+1. Create a distribution certificate in your Apple Developer Account.
+1. Create a distribution profile in your Apple Developer Account.
+1. Download provisioning profiles on your Mac.
+1. Add any required entitlements to your app.
+1. Add the code signing data to your app project.
+1. Validate package settings.
+1. Connect Visual Studio 2022 to a Mac build host.
+1. Publish your app using .NET CLI.
+
+## Create a certificate signing request
+
+To sign a .NET MAUI iOS app you must first create a certificate signing request (CSR) in Keychain Access on a Mac. For more information, see [Create a certificate signing request](provisioning.md#create-a-certificate-signing-request).
+
+## Create a distribution certificate
+
+The CSR allows you to generate a distribution certificate, which will be used to confirm your identity. The distribution certificate must be created using the Apple ID for your Apple Developer Account. For more information, see [Create a distribution certificate](provisioning.md#create-a-distribution-certificate).
+
+## Create a distribution profile
+
+To publish a .NET MAUI iOS app, you'll need to build a *Distribution Provisioning Profile* specific to it. This profile enables the app to be digitally signed for release so that it can be installed on an iOS device. A distribution provisioning profile contains an app ID and a distribution certificate. For more information, see [Create a distribution profile](provisioning.md#create-a-distribution-profile).
+
+## Download the provisioning profile on your Mac build host
+
+After creating the provisioning profile, it must be added to your Mac build host. The profile can be downloaded to your Mac build host with the following steps:
+
+1. On your Mac, launch Xcode.
+1. In Xcode, select the **Xcode > Preferences...** menu item.
+1. In the **Preferences** dialog, select the **Accounts** tab.
+1. In the **Accounts** tab, click the **+** button to add your Apple Developer Account to Xcode:
+
+    :::image type="content" source="media/overview/accounts-dialog.png" alt-text="Xcode Accounts dialog in preferences.":::
+
+1. In the account type popup, select **Apple ID** and then click the **Continue** button:
+
+    :::image type="content" source="media/overview/account-type.png" alt-text="Xcode select the type of account you'd like to add popup.":::
+
+1. In the sign in popup, enter your Apple ID and click the **Next** button.
+1. In the sign in popup, enter your Apple ID password and click the **Next** button:
+
+    :::image type="content" source="media/overview/sign-in.png" alt-text="Xcode Apple account sign-in.":::
+
+1. In the **Accounts** tab, click the **Manage Certificates...** button to ensure that your distribution certificate has been downloaded.
```

Comment:
Would it be wise to add something like: "no further manual action is needed for this. Just wait until the dialog shows and the download will be complete"?